sdk/translation/Azure.AI.Translation.Document/tests/TranslationOperationLiveTests.cs

```diff
@@ -431,8 +431,7 @@ public async Task DocumentTranslationWithGlossary()
             //changing the word First --> glossaryFirst and test --> glossaryTest
             string glossaryContent = "First, glossaryFirst\ntest, glossaryTest\n";
 
-            var glossaryContainerSasUri = await CreateGlossaryContainerAsync(new List<TestDocument> {new TestDocument (glossaryName, glossaryContent)});
-            Uri glossarySasUri = new Uri(String.Format("{0}{1}{2}{3}/{4}{5}", glossaryContainerSasUri.Scheme, Uri.SchemeDelimiter, glossaryContainerSasUri.Authority, glossaryContainerSasUri.AbsolutePath,glossaryName,glossaryContainerSasUri.Query));
+            var glossarySasUri = await CreateGlossaryAsync(new TestDocument (glossaryName, glossaryContent));
```

Comment:
love this new update where you moved the generation of the SAS to the `CreateGlossaryAsync` function